source/plugins/go/src/oms.go

```diff
@@ -1281,42 +1298,81 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 			msgpBytes = msgp.AppendMapStrStr(msgpBytes, fluentForward.Entries[entry].Record)
 		}
 
-		if MdsdMsgpUnixSocketClient == nil {
-			Log("Error::mdsd::mdsd connection does not exist. re-connecting ...")
-			CreateMDSDClient(ContainerLogV2, ContainerType)
-			if MdsdMsgpUnixSocketClient == nil {
-				Log("Error::mdsd::Unable to create mdsd client. Please check error log.")
-
+		if IsWindows == true {
```

Comment:
This condition should only be if Windows and Geneva log integration, sinec we havent merged Windows AMA

Reply:
reverted all the windows ama LA integration specific code.